minimal/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "react-native-fullstack-boilerplate",
+  "version": "0.1.0",
+  "main": "node_modules/expo/AppEntry.js",
+  "private": true,
+  "scripts": {
+    "start": "react-native-scripts start",
+    "exp:start": "exp start --lan",
+    "exp:android": "exp android",
```

Comment:
What is the difference between `exp android/ios` versus `react-native-scripts android/ios`?